<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HuboGlViewer</class>
 <widget class="QDialog" name="HuboGlViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="playBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QSlider" name="frameSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>351</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="glLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
